vizro-core/src/vizro/_vizro.py

```diff
@@ -49,6 +52,13 @@ def __init__(self, **kwargs):
             use_pages=True,
         )
 
+        # Ensure external_stylesheets is a list and append the additional stylesheet
+        external_stylesheets = self.dash.config.external_stylesheets
+        self.dash.config.external_stylesheets = (
+            external_stylesheets if isinstance(external_stylesheets, list) else [external_stylesheets]
+        )
```

Comment:
The check is helpful for when a user adds a link or links to the `external_stylesheets`. 

 In DMC there are variable to make it easy to add optional stylesheets.  For example,  `dmc.styles.DATES` is a single URL.  Or you can add all the optional stylesheets with `dmc.styles.ALL`  which is a list of links.   This has cause some confusion for DMC users.

Reply:
If you define `Vizro(external_stylesheets=dbc.themes.BOOTSTRAP).build(dashboard).run()`, `external_stylesheets `will also be of type `str`, and this code snippet would otherwise fail. So, contrary to what the Dash docs suggest, it's not always a list that is being returned

Reply:
You're right! Thanks for checking it 🚀